joplin/js/editor.js

```diff
@@ -4,13 +4,15 @@ import '../css/preview.scss';
 import insertWizardData from './CreateContentModal/insertWizardData';
 import menuActiveState from './EditPage/menuActiveState';
 import toggleActivePanel from './SidebarPreview/toggleActivePanel';
+import richTextPlaceholder from './CreateContentModal/richTextPlaceholder';
```

Comment:
`CreateContentModal` is a specific piece of the application. When you click to add a new page in Joplin, a Modal pops up that walks you through that process. So everything related to the CreateContentModal lives in the CreateContentModal/ folder. Your code would fit better in the EditPage/ folder because it affects the edit page only.

Reply:
good point! moved it there👍